miden-lib/src/accounts/faucets/mod.rs

```diff
@@ -33,8 +33,12 @@ pub struct BasicFungibleFaucet {
 impl BasicFungibleFaucet {
     // CONSTANTS
     // --------------------------------------------------------------------------------------------
-    const MAX_MAX_SUPPLY: u64 = (1 << 63) - 1;
-    const MAX_DECIMALS: u8 = 12;
+
+    /// The maximum value for the `max_supply` field.
+    pub const MAX_MAX_SUPPLY: u64 = (1 << 63) - 1;
+
+    /// The maximum number of decimals supported by the component.
```

Comment:
Is this enforced at the protocol-level? Because if so we could write something like this (same for above):
```suggestion
    /// The maximum number of decimals supported by faucets in the protocol.
```
cc @PhilippGackstatter @Fumuran

Reply:
`MAX_DECIMALS` is not enforced at the protocol level.